frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -41,14 +41,25 @@ class OverrideForkPointDialog(
 
   override fun createCenterPanel() = panel {
     row {
-      label(
-        format(
-          getString(
-            "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.HTML"
-          ),
-          branch.name
+      if (Regex("<.*>.*</.*>").containsMatchIn(branch.name)) {
```

Comment:
Maybe rather sth like
```suggestion
      if (escapeHtml4(branch.name) != branch.name) {
```